From ee1ed3c9a14a539ad37c6acd8644b90eb9c9cce0 Mon Sep 17 00:00:00 2001
Message-ID: <ee1ed3c9a14a539ad37c6acd8644b90eb9c9cce0.1699403993.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 23:55:27 +0000
Subject: [PATCH 65/67] gnu:
 go-github-com-go-telegram-bot-api-telegram-bot-api: Move to (gnu packages
 golang-web).

* gnu/packages/golang.scm (go-github-com-go-telegram-bot-api-telegram-bot-api): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Ia631773394faaff150c18b074c943565f6998e1f
---
 gnu/packages/golang-web.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm     | 25 -------------------------
 2 files changed, 25 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 41c8c37a0b..7759916c35 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -575,6 +575,31 @@ (define-public go-github-com-go-chi-chi-v5
 decompose request handling into many smaller layers.")
     (license license:expat)))
 
+(define-public go-github-com-go-telegram-bot-api-telegram-bot-api
+  (package
+    (name "go-github-com-go-telegram-bot-api-telegram-bot-api")
+    (version "4.6.4")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/go-telegram-bot-api/telegram-bot-api")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1x6j0k3aiicsr8l53na99ci10zm3qpn2syz4f60fzh164w5k1l7w"))))
+    (build-system go-build-system)
+    (home-page "https://go-telegram-bot-api.dev/")
+    (arguments
+     (list #:tests? #f                  ; Upstream tests are broken.
+           #:import-path "github.com/go-telegram-bot-api/telegram-bot-api"))
+    (propagated-inputs
+     (list go-github-com-technoweenie-multipartstreamer))
+    (synopsis "Golang bindings for the Telegram Bot API")
+    (description
+     "This package provides Golang bindings for the Telegram Bot API.")
+    (license license:expat)))
+
 (define-public go-github-com-goccy-go-json
   (package
     (name "go-github-com-goccy-go-json")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index d4d8d352a5..740ddbe440 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9921,31 +9921,6 @@ (define-public go-github-com-tekwizely-go-parsing
 parsers, and related tools.")
       (license license:expat))))
 
-(define-public go-github-com-go-telegram-bot-api-telegram-bot-api
-  (package
-    (name "go-github-com-go-telegram-bot-api-telegram-bot-api")
-    (version "4.6.4")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/go-telegram-bot-api/telegram-bot-api")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "1x6j0k3aiicsr8l53na99ci10zm3qpn2syz4f60fzh164w5k1l7w"))))
-    (build-system go-build-system)
-    (home-page "https://go-telegram-bot-api.dev/")
-    (arguments
-     (list #:tests? #f                  ; Upstream tests are broken.
-           #:import-path "github.com/go-telegram-bot-api/telegram-bot-api"))
-    (propagated-inputs
-     (list go-github-com-technoweenie-multipartstreamer))
-    (synopsis "Golang bindings for the Telegram Bot API")
-    (description
-     "This package provides Golang bindings for the Telegram Bot API.")
-    (license license:expat)))
-
 (define-public go-github.com-ulikunitz-xz
   (package
     (name "go-github.com-ulikunitz-xz")
-- 
2.41.0

